### PR TITLE
BUG: NiftiLabelsMasker inverse_transform without transform

### LINKED ISSUE #2192: NiftiLabelsMasker.inverse_transform() appears to fail in current nilearn version
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-79-642858f1debf> in <module>
----> 1 masker.inverse_transform(labels_and_my.values)

~/anaconda3/lib/python3.7/site-packages/nilearn/input_data/nifti_labels_masker.py in inverse_transform(self, signals)
    296         logger.log("computing image from signals", verbose=self.verbose)
    297         return signal_extraction.signals_to_img_labels(
--> 298             signals, self._resampled_labels_img_, self.mask_img_,
    299             background_label=self.background_label)

AttributeError: 'NiftiLabelsMasker' object has no attribute '_resampled_labels_img_'

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Get NiftiLabelsMasker.inverse_transform to work even without calling
transform.

Fixes #2192

Ping @banilo : this should fix your problem.

### CODE REVIEW

File: nilearn/input_data/nifti_labels_masker.py [hunk 1]
```diff
@@ -214,6 +214,9 @@ def fit(self, X=None, y=None):
 
             mask_data, mask_affine = masking._load_mask_img(self.mask_img_)
 
+        if not hasattr(self, '_resampled_labels_img_'):
+            self._resampled_labels_img_ = self.labels_img_
+
```
Comment (kchawla-pi) [inline nilearn/input_data/nifti_labels_masker.py]: Add a comment why we do this (transform now needed etc)

File: nilearn/input_data/tests/test_nifti_maps_masker.py [hunk 2]
```diff
@@ -115,11 +120,12 @@ def test_nifti_maps_masker():
                                         length=length)
     masker = NiftiMapsMasker(labels11_img, mask_img=mask21_img)
 
-    masker.fit_transform(fmri22_img)
+    signals = masker.fit_transform(fmri22_img)
     np.testing.assert_array_equal(masker._resampled_maps_img_.affine,
                                   affine2)
 
 
+
```
Comment (kchawla-pi) [inline nilearn/input_data/tests/test_nifti_maps_masker.py]: Flake8 complains about unused variables `fmri11_img_rr` & `signals`.

**Comment by kchawla-pi** [comment]
Plac8 Flake8 , add a comment, & this LGTM.

**Comment by GaelVaroquaux** [comment]
Good comments! And thanks for addressing them!!

**Comment by kchawla-pi** [comment]
I'll add a whats_new entry and then I'll merge this.

**Comment by GaelVaroquaux** [comment]
@kchawla-pi  Thanks for your work finishing this PR. It looks good. Given that @emdupre and you have previously approved this PR, I am taking the liberty of merging it.

**Pull request merged**